controllers/feeds/getFeedsV2.js

```diff
@@ -141,7 +149,7 @@ module.exports = async (req, res) => {
 
         const feeds = await getActivtiesOnFeed(feed, token, paramGetFeeds);
         if (feeds.length === 0) {
-          if (feed === 'main_feed') {
+          if (feed === 'main_feed_broad') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from checking `feed === 'main_feed'` to `feed === 'main_feed_broad'` seems fine as long as it aligns with the intended business logic. However, it's worth noting that hardcoding feed names like this can make the code less maintainable. If the feed names change in the future, you'll need to update them in multiple places. Consider defining these feed names as constants in a central location and referencing those constants throughout your code.

```diff
-  if (feed === 'main_feed_broad') {
+  if (feed === MAIN_FEED_BROAD) {
```
In the above snippet, `MAIN_FEED_BROAD` would be a constant defined elsewhere in your codebase.


<!-- This is an auto-generated comment by OSS CodeRabbit -->